ckanext/datastore/backend/postgres.py

```diff
@@ -1511,7 +1511,10 @@ def search_sql(context, data_dict):
         context['check_access'](table_names)
 
         for f in function_names:
-            if f not in backend.allowed_sql_functions:
+            for name_variant in [f, f.lower(), '"{}"'.format(f)]:
```

Comment:
Isn't is better to normalize the input function name using `format_entry()` in here as well?

Reply:
I think that would work, and might perhaps be slightly faster.